Gemfile.lock

```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     cocooned (2.2.1)
-      rails (>= 6.0, <= 7.2)
+      rails (>= 6.1, <= 7.3)
```

Comment:
@gael-ian I set this to `7.3` for consistency, but the next big release of Rails will be [Rails v8.0](https://github.com/rails/rails/milestones). I can change it to that if you'd like.